src/Event/ReadModel/JSONLD/EventLDProjector.php

```diff
@@ -275,6 +275,8 @@ protected function applyEventCreated(
 
         $jsonLD->attendanceMode = AttendanceMode::offline()->toString();
 
+        $jsonLD->typicalAgeRange = '-';
```

Comment:
It should also be set for events imported/updated from UDB2 I think?

Reply:
\+ Places imported/updated from UDB2 then

Reply:
Good point, I modified the XMLImporter to set all ages by default

Reply:
Looks good, I only think that places created via `PlaceImportedFromUDB2` should also get `"typicalAgeRange": "-"`?
It seems like the `CultuurNet\UDB3\Place\ReadModel\JSONLD\CdbXMLImporter` does not set any typicalAgeRange at first glance, probably because Actors did not have age info in UDB2. But we still want every place to have a `typicalAgeRange`, so we should also set the default for those.

Reply:
There is also an importer for places, I missed that one 🤦‍♂️